src/site/twitch.tv/modules/emote-menu/EmoteMenuTab.vue

```diff
@@ -29,6 +29,13 @@
 					>
 						<div class="set-sidebar-icon">
 							<img v-if="es.owner && es.owner.avatar_url" :src="es.owner.avatar_url" />
+							<div v-else-if="es.provider === 'EMOJI'" class="emoji-group">
+								<SingleEmoji
+									:id="emojiGroupIcons[emojiCategories.indexOf(es.name)]"
+									:alt="es.name"
+									:style="{ width: `100%`, height: `100%` }"
```

Comment:
The inline style may not be necessary as you already define a class in the outer div. Target this element within it

Reply:
I removed this styling in commit 1dda7d0